src/components/common/Extensions/UserNavigation/partials/UserReputation/UserReputation.tsx

```diff
@@ -0,0 +1,134 @@
+import clsx from 'clsx';
+import React, { FC, useEffect, useMemo, useState } from 'react';
+import { usePopperTooltip } from 'react-popper-tooltip';
+import {
+  useAppContext,
+  useColonyContext,
+  useDetectClickOutside,
+  useMobile,
+  usePrevious,
+  useUserReputation,
+} from '~hooks';
+
+import UserHub from '~common/Extensions/UserHub/UserHub';
+import Button from '~shared/Extensions/Button';
+import PopoverBase from '~shared/Extensions/PopoverBase/PopoverBase';
+import UserAvatar from '~shared/Extensions/UserAvatar';
+import MemberReputation from '~shared/MemberReputation/MemberReputation';
+import { UserReputationProps } from '../../types';
+import { transactionCount } from '~frame/GasStation/transactionGroup';
+
+export const displayName =
+  'common.Extensions.UserNavigation.partials.UserReputation';
+
+const UserReputation: FC<UserReputationProps> = ({
+  transactionAndMessageGroups,
+}) => {
+  const { colony } = useColonyContext();
+  const { wallet, user } = useAppContext();
+  const isMobile = useMobile();
+  const { profile } = user || {};
+  const [, setIsWalletButtonVisible] = useState(true);
+  const [isOpen, setOpen] = useState(false);
+  const [txNeedsSigning, setTxNeedsSigning] = useState(false);
+
+  const popperTooltipOffset = !isMobile ? [0, 8] : [0, 0];
+
+  const ref = useDetectClickOutside({
+    onTriggered: () => {
+      setOpen(false);
+      setTxNeedsSigning(false);
+    },
+  });
+  const { getTooltipProps, setTooltipRef, setTriggerRef, visible } =
+    usePopperTooltip(
+      {
+        delayShow: 200,
```

Comment:
add delayHide: 200 if you use interactive prop (it's recommended in documentation)